test/e2e/data/ironic-standalone-operator/operator/kustomization.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: ironic-standalone-operator-system
+
+resources:
+  - https://github.com/metal3-io/ironic-standalone-operator/releases/download/v0.5.1/install.yaml
+
+generatorOptions:
+  disableNameSuffixHash: true
+
+configMapGenerator:
+  - name: ironic-operator-config
+    literals:
+      - IPA_BASEURI=https://artifactory.nordix.org/artifactory/openstack-remote-cache/ironic-python-agent/dib
+
+patches:
+  - target:
+      kind: Deployment
+      name: ironic-standalone-operator-controller-manager
+    patch: |-
+      apiVersion: apps/v1
+      kind: Deployment
+      metadata:
+        name: ironic-standalone-operator
+      spec:
+        template:
+          spec:
+            containers:
+              - name: manager
+                envFrom:
```

Comment:
We need to bring this change to IrSO upstream, it's convenient to have a configmap even for production deploys.